src/main/java/frc/robot/hardware/rev/request/SparkMaxRequest.java

```diff
@@ -42,8 +44,12 @@ public T getSetPoint() {
 		return setPoint;
 	}
 
-	public Double getSetPointAsDouble() {
-		return setPointToDoubleConverter.apply(setPoint);
+	public Double getSetPointForSparkMAX() {
```

Comment:
I don't like the name ForSparkMAX because the entire class is for spark max, and the name doesn't tell me what the function does to the set point. I don't think I would understand what this function is without looking inside of it

Reply:
I think it's pretty clear, the forsparkmax returns the setpoint for the SparkMax, the setpoint that is entered to the motor

Reply:
U can do getSparkMaxCompatibleSetPoint or something like it if u want

Reply:
Funny but yeah sounds more right